scripts/sort-locales.js

```diff
@@ -0,0 +1,26 @@
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const fs = require("fs");
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const path = require("path");
+
+const directory = "src/Locale";
+
+fs.readdir(directory, (err, files) => {
+  if (err) throw err;
+
+  files.forEach((file) => {
```

Comment:
For performance we could have chosen just to do it for EN; others would be auto managed by crowdin; just a thought